setup-earthly/libs/setup-earthly.js

```diff
@@ -0,0 +1,36 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+const child_process = require('child_process');
+
+const IS_WINDOWS = process.platform === "win32";
+const IS_MAC = process.platform === "darwin";
+
+try {
+  console.log(`Setting up earthly`);
+
+  install_cmd = "sudo /bin/sh -c 'wget https://github.com/earthly/earthly/releases/latest/download/earthly-linux-amd64 -O /usr/local/bin/earthly && chmod +x /usr/local/bin/earthly'"
+  if (IS_WINDOWS) {
+    install_cmd = "sudo /bin/sh -c 'wget https://github.com/earthly/earthly/releases/latest/download/earthly-linux-amd64 -O /usr/local/bin/earthly && chmod +x /usr/local/bin/earthly'"
+  }
+  else if (IS_MAC) {
```

Comment:
```suggestion
  } else if (IS_MAC) {
```

Reply:
Perhaps we should add an `.eslint`

Reply:
sounds good, I created an Earthfile with a lint target for this.